graph-builder/Cargo.toml

```diff
@@ -41,6 +41,7 @@ custom_debug_derive = "^0.1.7"
 built = "^0.3.2"
 
 [dev-dependencies]
+cincinnati = { path = "../cincinnati", features = ["test"] }
```

Comment:
Not sure why are we enabling features here (also feature name is too generic imo)

Reply:
Certain test-related functions in the cincinnati crate are now gated by the `test` feature. This is a workaround to the limitation that code gated with `#[cfg(test)]` is not exposed to the tests of dependent crates. That's why I used `#[cfg(any(test, feature = "test"))]` to gate these functions in the cincinnati crate.

Reply:
Oh, I see, that makes sense